amethyst_renderer/src/pipe/effect/pso.rs

```diff
@@ -0,0 +1,165 @@
+use gfx::{handle, pso};
+use gfx::pso::{DataBind, DataLink, Descriptor, PipelineData, PipelineInit, InitError};
+use gfx::pso::buffer::{RawConstantBuffer, RawGlobal, RawVertexBuffer};
+use gfx::pso::resource::{RawShaderResource, Sampler};
+use gfx::pso::target;
+use gfx::shade::core::ProgramInfo;
+use types::{ColorFormat, DepthFormat, Resources};
+
+type AccessInfo = pso::AccessInfo<Resources>;
+type DepthStencilTarget = target::DepthStencilTarget<DepthFormat>;
+type Manager = handle::Manager<Resources>;
+type RenderTarget = target::RenderTarget<ColorFormat>;
+type RawDataSet = pso::RawDataSet<Resources>;
+type InitResult<'r, M> = Result<M, InitError<&'r str>>;
+
+#[derive(Clone, Debug, Default, Eq, Hash, PartialEq)]
+pub struct Meta {
+    const_bufs: Vec<RawConstantBuffer>,
```

Comment:
Constant buffers are not supported by OpenGL ES, how should we handle that? Also see https://github.com/amethyst/amethyst/issues/231